test/locale/fi.test.js

```diff
@@ -0,0 +1,27 @@
+import MockDate from 'mockdate'
+import dayjs from '../../src'
+import relativeTime from '../../src/plugin/relativeTime'
+import '../../src/locale/fi'
+
+dayjs.extend(relativeTime)
+
+beforeEach(() => {
+  MockDate.set(new Date())
+})
+
+afterEach(() => {
+  MockDate.reset()
+})
+
+it('Finnish locale relative time in past and future', () => {
+  expect(dayjs().add(1, 'd').locale('fi').fromNow())
+    .toBe('päivän kuluttua')
```

Comment:
it will be great if you could add moment.js result in test like: 

```
expect(dayjs().add(1, 'd').locale('fi').fromNow())
    .toBe('päivän kuluttua')
expect(dayjs().add(1, 'd').locale('fi').fromNow())
    .toBe(moment().add(1, 'd').locale('fi').fromNow())
```

Reply:
Those are actually not equal because the original translation here uses word ”kuluttua” and Moment ”päästä”. They are synonyms.

Reply:
The word used here is actually better but should they be the same?

Reply:
Yes, please. Day.js trying to keep everything the same with moment.js at the present. 

You can check other locales' test file as a reference.

Reply:
There seems to be another difference, too. Dayjs tells "2 päivää sitten" and Moment "kaksi päivää sitten". So for some reason, Moment uses words for numbers less than ten. Should this be replicated as well? This seems to be specific for Finnish locale although using integers would be fine.

Reply:
This difference has already been present here when the original Finnish translation is merged.

Reply:
I changed these.